R/tait-bryan.R

```diff
@@ -2,10 +2,19 @@
 #'
 #' aka euler angles
 #'
-#' @param yaw,pitch,roll Angles specified in radians
-#'
+#' @param yaw,pitch,roll angles
+#' @param unit Units of the angles, valid values are `"radians"` and `"degrees"`
 #' @export
-tait_bryan <- function(yaw, pitch, roll) {
+tait_bryan <- function(yaw, pitch, roll, unit = c("radians", 'degrees')) {
```

Comment:
While the primary use case is forcing people to specify radians or degrees, there is unexpected behavior if you e.g. say `unit="degree"` or `unit="aef"`. I think you'll probably want to use `match.arg` for this, like with the norms function (and include a test case to error when it's not specified properly)